.github/workflows/release.yaml

```diff
@@ -33,45 +33,36 @@ jobs:
           go-version: 'stable'
 
       # This step is only used for testing in the official kangaroo repo
-      - name: Overwrite Names for release testing and fetch kando webhapp from github
-        if: ${{ github.repository }} == 'holochain-apps/holochain-kangaroo-electron'
+      - name: Overwrite Names for release testing
+        if: ${{ github.repository }} == 'holochain-apps/kangaroo-electron'
         run: |
           echo ${{ github.repository }}
           echo "overwriting names for release testing"
-          curl -f -L --output ./pouch/ziptest.webhapp https://github.com/holochain/ziptest/releases/download/ziptest-v0.1.0/ziptest.webhapp
           node ./scripts/overwrite-with-test-name.js
 
       - name: Environment setup
         run: |
           yarn setup
 
-      - name: Retrieve version
-        run: |
-          echo "Retrieved App version: $(node -p -e "require('./package.json').version")"
-          echo "APP_VERSION=$(node -p -e "require('./package.json').version")" >> $GITHUB_OUTPUT
-        id: version
+      - name: Read kangaroo config
```

Comment:
The diffs are a bit confusing here unfortunately but it's essentially consolidating separate reads of the kangaroo config into a single step.